src/client/utils/tracking/trackOfferEvent.ts

```diff
@@ -96,13 +96,14 @@ export const useTrackOfferEvent = () => {
 
   const { quoteCartId } = useMatchedQuoteCartIdFromUrl()
 
+  const isDisabled = quoteCartId === undefined
   const [selectedInsuranceTypes] = useSelectedInsuranceTypes()
-
   const { data: quoteCartQueryData } = useQuoteCartQuery({
     variables: {
-      id: quoteCartId!,
+      id: quoteCartId || '',
       locale: isoLocale,
     },
+    skip: isDisabled,
   })
```

Comment:
This query will still be called many times and fail in the landing page and during the embark flow. Can we try to find a solution where we do not call it to start with?

Reply:
no, it will not be called, that's what skip param does :D